packages/ui/components/QuestionFormInput/components/RecallItemSelect.tsx

```
@@ -222,9 +225,11 @@
                     }
                   }}>
                   <div>{IconComponent && <IconComponent className="mr-2 w-4" />}</div>
-                  <div className="max-w-full overflow-hidden text-ellipsis whitespace-nowrap text-sm">
-                    {getRecallLabel(recallItem.label)}
-                  </div>
+                  <div
+                    dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)